### DISCUSSION TITLE
OPENMETADAT CONNECT ORACLE ERROR: ORA-12541: TNS:no listener

### DISCUSSION THREAD

**Discussion by TheReasonMilami** [open-metadata/OpenMetadata#18356] [Q&A]
I establishing a connection to oracle 19c with open metadata but i got error: ORA-12541: TNS:no listener.
I filled:
`Host and Port` : `host.docker.internal:1521`
`Oracle connection type`: `Oracle TNS connection`
`Oracle TNS Connection String`: `(DESCRIPTION =     (ADDRESS = (PROTOCOL = TCP)(HOST = 127.0.0.1)(PORT = 1521))     (CONNECT_DATA =       (SERVER = DEDICATED)       (SERVICE_NAME = orclpdb)     )   )`
`Oracle instant client directory`: `D:/file_exe/WINDOWS.X64_193000_db_home/instantclient`
`Database Name`: `orclpdb`
`connection schema`: `oracle+cx_oracle`

I kept getting the error ORA-12541. `HOST` in file `listener.ora` and `tnsnames.ora` all are `127.0.0.1`. I checked with `lsnrctl status`, database still work 
![image](https://github.com/user-attachments/assets/75b4ff21-2407-480c-9821-9bba854ec217)

My sql developer can still connect to database with `tns` connection.
What is exactly problem in my case? Thanks for every answers.

**Comment by Prajwal214** [comment]
@TheReasonMilami Can you use host.docker.internal as host under `Oracle TNS Connection String`
